### DISCUSSION TITLE
Scaling preset

### DISCUSSION THREAD

**Discussion by MikuOgiwara** [ppy/osu#28217] [Ideas] [answered]
For Taiko players, we frequently like to change resolutions. However, in Lazer, we cannot change the resolution in windowed mode (which is annoying), and in full-screen mode, Lazer changes the screen resolution directly, causing issues on Windows when you quit lazer.
Exemple of the issue:
![331860049-54d538f5-4007-4898-9bc8-c663e5ba0bdd](https://github.com/ppy/osu/assets/108834620/8b95a8e3-91cb-4d37-9919-fd9410496933)

The current solution is to play in full-screen mode with scaling, but since scaling doesn't have presets, it is difficult to change resolutions quickly.

Therefore, would it be possible to have resolution presets for scaling as well as for windowed mode? Thank you very much.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/17999
